source/IO/dataset_import/AIRRImport.py

```diff
@@ -21,61 +78,84 @@ class AIRRImport(DataImport):
         my_airr_dataset:
             format: AIRR
             params:
-                # required parameters:
-                metadata_file: path/to/metadata.csv
-                path: path/to/directory/with/repertoire/files/
-                result_path: path/where/to/store/imported/repertoires/
-                # optional parameters (if not specified the values bellow will be used):
-                import_productive: True # whether to import productive sequences or not to import them
-                import_with_stop_codon: False # whether to import sequences with stop codon
-                import_out_of_frame: False # whether to import sequences which are out of frame (where vj_in_frame is False)
-                columns_to_load: [sequence_aa, sequence, v_call, j_call, locus, duplicate_count, productive, vj_in_frame, stop_codon] # to import other columns, add them to this list
-                column_mapping: # AIRR column names -> immuneML repertoire fields
+                path: path/to/files/
+                is_repertoire: True # whether to import a RepertoireDataset (True) or a SequenceDataset (False)
+                metadata_file: path/to/metadata.csv # metadata file for RepertoireDataset
+                metadata_column_mapping: # metadata column mapping AIRR: immuneML for SequenceDataset
```

Comment:
I chose to write it like this (without arrow) because the user has to write it with just a colon. The user doesn't need to know how we use the mapping specifically, we could have implemented it in another way, they just need to know on which side of the colon to put the fields. But if you prefer with arrow I'm willing to add it, I have no strong opinion.

Reply:
i didn't mean the arrow, just that they know that on the left side is the AIRR name and on the right the immuneML name but i see now that it's already there in a way though maybe not super clear? Maybe sth like "metadata column mapping for AIRR fields to immuneML fields" (there are probably better words for this, but this is the idea)?

Reply:
It is explained more extensively under 'Arguments', I tried to keep the comments in the yaml example short, as they are more for a person who already is familiar with import and wants to take a quick look without reading too much